pages/admin.js

```diff
@@ -1,8 +1,13 @@
 import Dashboard from "../components/admin_real/shared/Dashboard";
+import ReservationsContextProvider from "../components/shared/ReservationsContextProvider";
 
 // Example: https://github.com/vercel/nextjs-subscription-payments
 // Current Issue: https://github.com/supabase/auth-helpers/issues/281
 
 export default function Admin() {
-  return <Dashboard />;
+  return (
+    <ReservationsContextProvider>
```

Comment:
This is not the right place for this, but we can leave it for now. Ideally this is wrapped around the calendar and the table individually.

Imagine in the future we add a page we don't want to add this functionality, this will break